core/src/voice_fn/processors/openai.clj

```diff
@@ -195,30 +194,43 @@
               :else
               (recur function-name function-arguments tool-call-id))))))))
 
-(defmethod pipeline/create-processor :processor.llm/openai
-  [id]
-  (reify p/Processor
-    (processor-id [_] id)
-
-    (processor-schema [_] OpenAILLMConfigSchema)
-
-    (accepted-frames [_] #{:frame.context/messages :frame.control/interrupt-start})
-
-    (make-processor-config [_ _ processor-config]
-      (m/decode OpenAILLMConfigSchema processor-config mt/default-value-transformer))
-
-    (process-frame [_ pipeline processor-config frame]
-      (cond
-        (and (frame/llm-context? frame)
-             ;; LLM shouldn't respond to itself
-             (not (u/assistant-last-message? (:frame/data frame)))
-             (not (pipeline/interrupted? @pipeline)))
-        (process-context-frame id pipeline processor-config frame)
-
-        ;; If interrupt-start frame is sent, we cancel the current token
-        ;; generation if one is in progress
-        (frame/control-interrupt-start? frame)
-        (when-let [stream-ch (get-in @pipeline [id :stream-ch])]
-          ;; Closing the stream-ch, prevents new events to be put on it
-          (a/close! stream-ch)
-          (swap! pipeline update-in [id] dissoc :stream-ch))))))
+(def openai-llm-process
+  (flow/process
+    {:describe (fn [] {:ins {:in "Channel for incoming context aggregations"}
+                       :outs {:out "Channel where streaming responses will go"}})
+     :parmas {:llm/model "Openai model used"
```

Comment:
`:params`

Reply:
Fixed. Thank you for the review, Sean! 🙏